packages/sdk/src/application/ApplicationClient.ts

```diff
@@ -438,31 +437,55 @@ export class ApplicationClient extends APIResource {
         await this._client.post<RestartPlaygroundConversationResponse>(
           queryString,
         );
-      return response;
+      return Promise.resolve(response);
     } catch (error) {
       return Promise.reject(error);
     }
   }
 
-  async chat(props: ChatRequest): Promise<ChatResponse> {
-    const { ownerId, appId, ...payload } = props;
+  async chat(
+    props: ChatRequest,
+    stream: boolean = true,
+  ): Promise<ReadableStream<Uint8Array>> {
+    const { ownerId, appId, requesterUid, accessToken, ...payload } = props;
+
+    if (
+      !accessToken ||
+      !ownerId ||
+      !appId ||
+      !payload.catalogId ||
+      !payload.conversationUid
+    ) {
+      throw new Error("Required parameters are missing");
```

Comment:
@thewbuk Please imagine yourself is the user, you can't debug this error when all you get is the same message, try to get more hint to the user